types/material-ui/index.d.ts

```diff
@@ -1,6 +1,6 @@
-// Type definitions for material-ui v0.16.0
+// Type definitions for material-ui v0.16.1
```

Comment:
So the version should be the same version of the library that you are trying to type. I understand that not all 0.17.1 change in typing is in. But I think it will be more clear if we use 0.17.1 and then in the future people can come in and patch the rest. 

Note: I intentionally use 0.17.1 instead of 0.17.2 as @oliviertassinari  pointed out that this property is add in 0.17.1